### ISSUE TITLE
ME player simulator

### ISSUE THREAD

**Issue by Mecr0mage** [AppliedEnergistics/Applied-Energistics-2#6787]
### Describe the feature

A ME part that can be set to do a specific action that mimics the Player either with an empty hand or a specific item (e.g. left mouse, right mouse) and can have a tick delay set. Redstone control upgradeable for autocrafting

system could be expanded to potentially have a whole pattern of actions, possibly with multiple items, encoded for more advanced interactions. if there is time and/or a want

### Reasons why it should be considered

for one this would allow automation of XP gathering from mob farms as well as a way to automated the usage of looting and other enchantments on those mob farm.
this would also allow the automation of some parts of mods that need physical interaction to produce an effect, and if the simulated player action acts as the player who places it, it might be able to use whatever mana/power function some magic mods add to players, or other similar player specific mechanics.

**Comment by Sea-Kerman** [comment]
yeah, an item user would make the autocrafting capabilities much more complete, especially one which could simulate a series of clicks at different angles

**Comment by Technici4n** [comment]
I don't think this fits with the rest of the mod.
